src/Microsoft.Azure.AppConfiguration.AspNetCore/AzureAppConfigurationRefreshMiddleware.cs

```diff
@@ -16,26 +14,12 @@ namespace Microsoft.Azure.AppConfiguration.AspNetCore
     class AzureAppConfigurationRefreshMiddleware
     {
         private readonly RequestDelegate _next;
-        public IList<IConfigurationRefresher> Refreshers { get; private set; }
+        public IEnumerable<IConfigurationRefresher> Refreshers { get; }
```

Comment:
Does that thing need to be public?

Reply:
The class itself is `internal`, so we follow the convention of keeping members `public`. Also, the unit tests seem to fail to access `internal` property within `internal` class but can access `public` properties within `internal` class fine with the `InternalsVisibleTo` configuration.

Reply:
I see. I'm not aware of any convention of keeping members public in internal classes. But if we have it for the tests to pass then I'm fine.

Reply:
Oh, I see you meant keeping them public rather than internal. And on top of that the reason it is public is for the unit tests. Got it.